build.gradle.kts

```diff
@@ -34,3 +37,7 @@ tasks {
         gradleVersion = "7.4"
     }
 }
+
+rootProject.plugins.withType<NodeJsRootPlugin> {
+    rootProject.the<NodeJsRootExtension>().nodeVersion = "16.14.0"
```

Comment:
Revert it please.
Kotlin is already updated to 1.6.20-M1 with new Node.js onboard.